Composer/packages/server/src/controllers/orchestrator.ts

```diff
@@ -14,6 +15,8 @@ enum DownloadState {
   DOWNLOADING,
 }
 
+let errorMsg: any;
```

Comment:
why is this variable defined in the global context? I suggest defining it in a closure context.

Reply:
wrapped the controller in a class to address this